lib/Configuration.php

```diff
@@ -99,7 +99,7 @@ class Configuration {
 	 *
 	 * @var string
 	 */
-	protected $userAgent = 'OpenAPI-Generator/1.0.0/PHP';
+	protected $userAgent = 'BHR-SDK/2.0.0/PHP';
```

Comment:
@drago1216 this is the main thing in this PR - updating the user agent to something meaningful so it's easy to categorize in datadog